src/style.css

```
@@ -1 +1,42 @@
 /* add styles here */
+* body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--2 > .stars__star:nth-child(-n + 2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 > .stars__star:nth-child(1) {
+  background-image: url(./images/star-active.svg);
```

combine all these styles with comma
```suggestion
.stars--2 > .stars__star:nth-child(-n + 2),
.stars--1 > .stars__star:nth-child(1),
..... {
  background-image: url(./images/star-active.svg);
```